src/aggregation/segment_agg_result.rs

```diff
@@ -0,0 +1,35 @@
+use std::collections::HashMap;
+
+use crate::DocId;
+
+use super::metric::AverageData;
+
+pub type AggregationResultTree = HashMap<String, AggregationResult>;
+
+pub enum AggregationResult {
```

Comment:
you already have a file with aggregation result... Is there something specific at the aggregation level?